spec/factories/users.rb

```diff
@@ -19,7 +19,7 @@
 
 FactoryBot.define do
   factory :user do
-    password = Faker::Internet.password
+    password = Faker::Internet.password(6, 12)
```

Comment:
I had looked for that but thought with devise it was too complex to test, is it really possible?

Reply:
Yup